url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gcloud/GCloudUrlUpdater.java

```diff
@@ -0,0 +1,46 @@
+package com.devonfw.tools.ide.url.updater.gcloud;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.WebsiteUrlUpdater;
+
+import java.util.regex.Pattern;
+
+public class GCloudUrlUpdater extends WebsiteUrlUpdater {
+
+  @Override
+  protected String getTool() {
+
+    return "gcloud";
+  }
+
+  @Override
+  protected void addVersion(UrlVersion urlVersion) {
+
+    String baseUrl = "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${version}-${os}-";
+
+    doAddVersion(urlVersion, baseUrl + "x86_64.zip", WINDOWS);
+    //TODO add Linux and MacOS
+  }
+
+  @Override
+  protected String getVersionUrl() {
+
+    return "https://cloud.google.com/sdk/docs/release-notes";
+  }
+
+  @Override
+  protected Pattern getVersionPattern() {
+
+    return Pattern.compile("(\\d+\\.\\d+\\.\\d+)");
+  }
+
+  @Override
+  protected String mapVersion(String version) {
+
+    if (version.matches("\\d{3,}\\.\\d+\\.\\d+")) {
```

Comment:
There are many releases for `gcloud`. In my opinion we should just provide versions higher `100.x.x` or even `200.x.x`. We could also use the regex on the `getVersionpattern()` function and do without mapping

Reply:
In such case, it would be better to require a minimum version. See example here:
https://github.com/devonfw/ide/blob/master/url-updater/src/main/java/com/devonfw/tools/ide/url/updater/node/NodeUrlUpdater.java#L48

Reply:
Done